addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -216,13 +216,7 @@ def _get_livechat_discuss_channel_vals(
         # partner to add to the discuss.channel
         operator_partner_id = user_operator.partner_id.id if user_operator else chatbot_script.operator_partner_id.id
         members_to_add = [
-            Command.create({
-                # making sure the unpin_dt is always later than the last_interest_dt
-                # so that the channel is always unpinned at first
-                'last_interest_dt': fields.Datetime.now() - timedelta(seconds=30),
```

Comment:
Note: not directly related to the PR, but I fixed the default of `last_interest_dt` to remove this annoying trick (as not fixing it made the test harder to write).